m02_setup/01_first_steps.rst

```diff
@@ -0,0 +1,190 @@
+.. The roles  should be created at the beginning of the document. Global roles can be created in the `global.rst` file
+
+.. include:: /global.rst
```

Comment:
This "include global.rst" line is removed if you use the conf.py (`rst_prolog`) for it.